python/matmul/test.py

```diff
@@ -42,14 +42,14 @@ def __init__(self, tiling_transforms):
                                                    peel=[])
 
 # 1 level of tiling and then generalize and interchange.
-expert_tile_1_and_generalize_interchange = Tiling.then(Generalization).then(
-    Vectorization).then(LoweringOnlyExpert)('matmul_on_tensors',
-                                            'linalg.generic',
-                                            tile_sizes=[8, 8, 24],
-                                            tile_interchange=[2, 0, 1],
-                                            pad=False,
-                                            peel=[],
-                                            iterator_interchange=[0, 2, 1])
+expert_tile_1_and_generalize_interchange = Tile.then(Generalize).then(
```

Comment:
Nite: can we make this resemble:

```
expert_tile_1_and_generalize_interchange = \
  Tile('matmul_on_tensors',                              \
         'linalg.generic',                                          \
         tile_sizes=[8, 8, 24],                                 \
         tile_interchange=[2, 0, 1],                        \
         pad=False,                                                 \
         peel=[],                                                      \
         iterator_interchange=[0, 2, 1])                \
  .then(Generalize)                                             \
  .then(Vectorize)                                               \
  .then(LoweringOnlyExpert)
```

or is python going to complain?

Reply:
yes, I insist on the backslashes being at exactly those positions :) ... sigh webUI

Reply:
We can't (yet) have parameterized transforms followed by unparameterized transforms, so no options directly to the first `Tile`. Added the backslashes though, I would have hoped for the formatter to put them in.

Reply:
sg, thanks!